src/components/property/overview/property-card.tsx

```diff
@@ -42,6 +42,9 @@ const PropertyCard = ({ property, handleDelete, rentalUnits, tenancies }: Proper
     const vacancies = runningTs?.filter((t) => t.isVacancy);
     const numOfVacancies = vacancies?.length ? vacancies.length : 0;
 
+    const fallbackImageUrl =
+        'https://firebasestorage.googleapis.com/v0/b/immo-manager.appspot.com/o/images%2Fproperties%2Fproperty-thumbnail-placeholder.jpg?alt=media&token=ccb13819-d815-4814-9e36-017c33838d10';
+
```

Comment:
das funktioniert nicht mit dem token in der url soweit ich sehen kann

Reply:
Hmm bei mir schon, habs sogar mit dem Inkognito-Modus getestet.
Oder meinst du, man sollte das nicht dort drin stehen haben? Kommt man sonst alternativ an eine Url ohne Token?

Reply:
Ich habe erneut versucht das bild zu öffnen und jetzt hat es funktioniert! sry kA woran es vorher lag, gab einen error. Aber grundsätzlich finde ich es besser default bilder lokal im asset folder abzulegen, dann gibt es auch keine dependency.

Reply:
Aber koennen wir ja auch irgendwann noch nachholen.

Reply:
Das habe ich eben auf die schnelle nicht hinbekommen. Mache sonst grad ein Ticket dazu